mountpoint-s3/tests/cli.rs

```diff
@@ -170,3 +170,65 @@ fn invalid_profile() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn validate_default_log_files_permissions() -> Result<(), Box<dyn std::error::Error>> {
+    let mut cmd = Command::cargo_bin("mount-s3")?;
+    let dir = assert_fs::TempDir::new()?;
+    let default_log_dir = home::home_dir().unwrap().join(".mountpoint-s3");
+
+    cmd.arg("test-bucket").arg(dir.path());
+    cmd.assert().failure();
```

Comment:
Why failure?

I'm wondering if we're better using a known bucket like `doc-example-bucket` or the forbidden bucket?

Reply:
Actually, we don't care what Mountpoint returns here as long as the logs are printed out to the file, so it could be either success or failure. In this case, it's just easier to make Mountpoint fail and not having to cleanup the mount after test.

Reply:
Also, the tests here are unit tests so we don't really want to mount to a real bucket, but I can change the bucket name to `doc-example-bucket` if we think that's better.

Reply:
Yeah, let's use a bucket we know can't exist, which is `DOC-EXAMPLE-BUCKET`.